src/official-account/official-account.ts

```diff
@@ -1,36 +1,40 @@
 /* eslint-disable camelcase */
 import { EventEmitter } from 'events'
+import { v4 } from 'uuid'
 
 import crypto from 'crypto'
-import { ContactGender, FileBox, log }  from 'wechaty-puppet'
+import { ContactGender, FileBox, log } from 'wechaty-puppet'
 import { FileBoxType } from 'file-box'
 
 import { normalizeFileBox } from './normalize-file-box'
 
 import {
   Webhook,
   VerifyArgs,
-}                      from './webhook'
+} from './webhook'
 import { PayloadStore } from './payload-store'
 import {
   getSimpleUnirest,
   SimpleUnirest,
-}                       from './simple-unirest'
+} from './simple-unirest'
```

Comment:
Could we keep the spaces for better readability? Because I feel that each import should be a separate block visually.

Reply:
```typescript
import crypto from 'crypto'
import { ContactGender, FileBox, log } from 'wechaty-puppet'
import { FileBoxType } from 'file-box'

import { normalizeFileBox } from './normalize-file-box'
```

the indent of `from` keyword should be keep the same indent in the whole file ?

```typescript
import crypto                           from 'crypto'
import { ContactGender, FileBox, log }  from 'wechaty-puppet'
import { FileBoxType }                  from 'file-box'

import { normalizeFileBox }             from './normalize-file-box'
```

should we keep this indent style ?

Reply:
> the indent of from keyword should keep the same indent in the whole file?

We can define three rules for the code style of the `import` block:

1. One line should only include a maximum of one imported component
1. the `from` should ident spaces until it will in the right of all the imported components;
1. keep the same indent of `from` in one block unless there's a blank line separate the blocks.

The following example from your comment will satisfy the above 3 rules:

```ts
import crypto          from 'crypto'
import {
  ContactGender,
  FileBox,
  log,
}                      from 'wechaty-puppet'
import { FileBoxType } from 'file-box'

import { normalizeFileBox } from './normalize-file-box'
```